libmscore/repeatlist.cpp

```diff
@@ -415,7 +425,7 @@ void RepeatList::unwindSection(Measure* sectionStartMeasure, Measure* sectionEnd
                   }
             if (doJump && !isGoto) {
                   Jump* s = 0;
-                  foreach(Element* e, m->el()) {
+                  foreach (Element* e, m->el()) {
```

Comment:
yes, but we just change them as we touch the neighbouring code,
Adding this to the developers handbook / coding stly makes sense to me ;-)